velox/functions/prestosql/tests/ArrayRemoveTest.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::exec;
+using namespace facebook::velox::functions::test;
+
+namespace {
+
+class ArrayRemoveTest : public FunctionBaseTest {
```

Comment:
Can we add some tests to cover the following cases:
```SQL
SELECT ARRAY_REMOVE(ARRAY[1, 1, NULL, 2, 3], NULL);
SELECT ARRAY_REMOVE(ARRAY[ARRAY[1, 1], NULL, ARRAY[2, 2]], ARRAY[1, 2]);
```

Reply:
Thank your comments.
that's a good idea, I'll add these two ut later

Reply:
@zacw7 I have added these two tests. Thanks for your suggests.